plugins/core/Standard/qFacets/include/qFacets.h

```diff
@@ -45,6 +48,7 @@ class StereogramDialog;
 //! Facet detection plugin (BRGM)
 /** BRGM: BUREAU DE RECHERCHES GEOLOGIQUES ET MINIERES - http://www.brgm.fr/
 **/
+
```

Comment:
empty line to be removed (doxygen documentation should stick with the class/object)